RadixWallet/Clients/RadixConnectClient/RadixConnectClient+Live.swift

```diff
@@ -14,25 +14,6 @@ extension RadixConnectClient {
 		let radixConnectMobile = RadixConnectMobile()
 		let localNetworkAuthorization = LocalNetworkAuthorization()
 
-		Task {
```

Comment:
Don't start the tasks as soon as the Client is instantiated, instead these will be started on main screen, after the profile is available.